config/config.go

```diff
@@ -0,0 +1,35 @@
+package config
+
+import (
+	"github.com/BurntSushi/toml"
+	log "github.com/Sirupsen/logrus"
```

Comment:
This is a convention I've seen across a bunch of apps (regardless of what library you use, import it as log).